lib/dataLayer.tsx

```diff
@@ -207,11 +207,11 @@ async function _submitToApI(
             : "";
         router.push(
           {
-            pathname: `/${language}/confirmation`,
+            pathname: `/id/${formMetadata.id}/confirmation`,
             query: { ...referrerUrl, htmlEmail: htmlEmail, pageText: endPageText },
           },
           {
-            pathname: `/${language}/confirmation`,
+            pathname: `/id/${formMetadata.id}/confirmation`,
```

Comment:
Thinking you meant to modify this line to `/${language}/id/${formMetadata.id}/confirmation` to render `/en/id/2/confirmation`